webapp/components/setting_picture.jsx

```diff
@@ -190,7 +196,11 @@ export default class SettingPicture extends Component {
                                 errors={[this.props.clientError, this.props.serverError]}
                                 type={'modal'}
                             />
-                            <span className='btn btn-sm btn-primary btn-file sel-btn'>
+                            <button
+                                className='btn btn-sm btn-primary btn-file sel-btn'
+                                disabled={fileInputDisabled}
+                            >
+                                {selectButtonSpinner}&nbsp;
```

Comment:
Can you use CSS to space this out instead of using an nbsp so that we have some more control over its appearance?